_posts/2017-03-16-two-months-of-tech-interviews.md

```diff
@@ -0,0 +1,220 @@
+1. Hiring decision
+
+When I accepted my job offer, I was in the middle of a few other processes. I ended up having to withdraw my applications (not enough time to finish 😥). When I was applying, I tried to avoid this as much as possible by spacing out my interviews. 
+
+Here's how I progressed through my interviews:
+
+![Progress through various interview processes](/assets/posts/interview-pipeline.png){: .center-image }
+
+### Interview challenges
+
+Interviewing was a bunch of ups and downs.
+
+Some days I'd be applying all over and feeling super excited about my chances.  
+Other days I'd be freaking out because weeks had gone by without any responses.
+
+Some days I'd be scheduling interviews non-stop.  
+Other days I was looking back on rejections and feeling defeated by all the failures.
+
+Some days I felt bad about my interviews but then got forwarded along.  
+Other days I felt great about my interviews and then got rejected.
+
+It's also a well known fact that interviewing is not an exact science. 
+
+After my first couple of rejections, I started getting really frustrated with technical interviews.
+
+> These coding problems are so arbitrary! If I ever needed to know this, I could easily Google it.
+
+> I'm not allowed to run the code? How is this testing my actual abilities?
+
+> Yikes! I've been doing nothing but algorithm prep for a month and this is a CSS question...
+
+I wished people would just ask me about my past projects and how I work with other people.
+
+Then I had some really laid back interviews that did just that - those left me concerned.
+
+> That was all conversational... what if they just hire random people who can talk but can't code?!
+
+> He didn't write any notes. What's he going off of other than just gut instinct?
+
+> This is too easy...
+
+By the end I had a new appreciation for how hard it is to interview.
+
+- You can keep things really conversational, but how does that show if someone can actually code?
+- You can assign a project, but how can you know if the person will have time to do a good job?
+- You can dive into quick algorithm problems, but how much does that actually tell you about the person?
+
+All the while you have a growing stack of applicants just waiting for an interview. 😩
+
+I wonder if any companies let candidates choose. Give a few different options and let the candidate choose the path that works best for them. I'd like to see how that type of system would work out.
+
+### Interview highlights
+
+Luckily, not all my interviews left me banging my head against a wall. Some were really fun! 
+
+- **HackerNews MVP:** One company asked me to build an MVP for HackerNews in 4 hours. I really enjoyed being able to focus on a project without having an interviewer breathing down my neck. The followup was working with some team members to extend the MVP. Since it was my code, I became the most knowledgeable person. Being the one giving directions made the interviewers less intimidating.
```

Comment:
Sounds like a good experience. I would've been scared by this a bit. Four hours sound like a very short time. I'd thus be interested in how well they defined the exercise: Was it only this one sentence ("build MVP for HN") or were there more requirements given?